pkgs/applications/science/biology/minc-widgets/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake, makeWrapper,
+  perl, GetoptTabular, MNI-Perllib,
+  libminc, octave, coreutils, minc_tools }:
+
+stdenv.mkDerivation rec {
+  pname = "minc-widgets";
+  name  = "${pname}-1.0.0";
+
+
+  src = fetchFromGitHub {
+    owner  = "BIC-MNI";
+    repo   = pname;
+    rev    = "f08b643894c81a1a2e0fbfe595a17a42ba8906db";
+    sha256 = "1b9g6lf37wpp211ikaji4rf74rl9xcmrlyqcw1zq3z12ji9y33bm";
+  };
+
+  nativeBuildInputs = [ cmake makeWrapper ];
+  buildInputs = [ libminc ];
+  propagatedBuildInputs = [ perl GetoptTabular MNI-Perllib octave coreutils minc_tools ];
+
+  postFixup = ''
+    for p in $out/bin/*; do
+      wrapProgram $p --prefix PERL5LIB : $PERL5LIB --set PATH "${stdenv.lib.makeBinPath [ coreutils minc_tools ]}";
+    done
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/BIC-MNI/${pname}";
+    description = "Collection of Perl and shell scripts for processing MINC files";
+    maintainers = with maintainers; [ bcdarwin ];
+    platforms = platforms.unix;
+  };
```

Comment:
For these widgets, ecat2hdf5, mincanon and volunfold seem to have no license or copyright information with them. (And otherwise there seems to no general license information for the whole repository.) So in this case they should default to `licenses.unfree`. All the other widgets seem to have that same free software license as with the previous packages. However, and I'm no lawyer, but to stay safe I think we should default to a `licenses.unfree` for the entire package, just because those aforementioned 3 widgets are nonfree. So put `license = licenses.unfree;`. :+1: